pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -2059,3 +2040,34 @@ func InterpretTransitionalModelType(useVirtioTransitional *bool, archString stri
 	}
 	return "virtio-non-transitional"
 }
+
+func DomainVCPUTopologyForHotplug(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
```

Comment:
Do we need this to be exported?

Reply:
You are correct, its redundant. will fix it